mobify/spline
# Prepare Spline for NPM

Reviewers: @marlowpayne, @mikenikles 
CC: @yourpalsonja @cole-sanderson @mlworks @Magitron

## Changes
- Remove Bower as the distribution network
- Prepare this repo for distribution on NPM

## Todos:
- [ ] Add missing utility mixins to be used in Progressive Web SDK and projects (scaffold)
- [ ] +1 Engineer

## How to Test
- Testing the Repo itself
    - Follow the "How To Test" instructions in the readme
- Test in [SDK](https://github.com/mobify/progressive-web-sdk)
    - Clone the SDK repo and run `npm install`
    - In one of the SDK components, add `@import node_modules/spline/spline`
    - Apply one of Spline's mixins to the component SCSS
    - Run `npm run styleguide`
    - Open [http://0.0.0.0:4000](http://0.0.0.0:4000) and find the component you tweaked
    - Confirm that the mixin styles were applied
- Test in [Scaffold](https://github.com/mobify/progressive-web-scaffold)
    - Clone the scaffold repo and run `npm install`
    - In one of the Button component, add `@import node_modules/spline/spline`
    - Apply one of Spline's mixins to the component SCSS
    - Run `npm run styleguide`
    - Open [http://0.0.0.0:4000](http://0.0.0.0:4000) and find the component you tweaked
    - Confirm that the mixin styles were applied
